tembo-cli/src/cmd/init.rs

```diff
@@ -1,16 +1,32 @@
 use crate::cli::context::{
     tembo_context_file_path, tembo_credentials_file_path, tembo_home_dir, CONTEXT_DEFAULT_TEXT,
-    CREDENTIALS_DEFAULT_TEXT,
+    CONTEXT_EXAMPLE_TEXT, CREDENTIALS_DEFAULT_TEXT, CREDENTIALS_EXAMPLE_TEXT,
 };
 use crate::cli::file_utils::FileUtils;
 use crate::tui::confirmation;
 use clap::Args;
+use std::env;
 
 /// Initializes a local environment. Creates a sample context and configuration files.
 #[derive(Args)]
 pub struct InitCommand {}
 
 pub fn execute() -> Result<(), anyhow::Error> {
+    // Determine if running tests
+    let is_test_env = env::var("RUNNING_TESTS").is_ok();
+
+    let context_text = if is_test_env {
+        CONTEXT_EXAMPLE_TEXT
+    } else {
+        CONTEXT_DEFAULT_TEXT
+    };
+
+    let credentials_text = if is_test_env {
+        CREDENTIALS_EXAMPLE_TEXT
+    } else {
+        CREDENTIALS_DEFAULT_TEXT
+    };
```

Comment:
Could you tell me how I could approach this problem for the tests since it wouldn't run without any profiles set up? I don't think I could run login because it opens up mahout which won't work in github actions